agent/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM kubeshop/tracetest:v0.13.9
```

Comment:
this looks like a very easy thing to forget to update on every release. can we make it dynamic somehow?

Reply:
do you think it's worth adding it to the pipeline now and make it build the same way our main image?

Reply:
yea, totally, no time like the present

Reply:
Done!